tests/ludwig/config_validation/test_checks.py

```diff
@@ -17,6 +17,23 @@
 from tests.integration_tests.utils import binary_feature, text_feature
 
 
+def test_passthrough_number_decoder():
+    config = {
+        "defaults": {"number": {"decoder": {"fc_norm": None, "fc_output_size": 10, "type": "passthrough"}}},
```

Comment:
Do we want to test setting the passthrough decoder in the output feature as well? I know the feature configs get populated with the defaults behind the scenes, but that could be a nice sanity check.

Reply:
added!